framework/pom.xml

```diff
@@ -9,27 +9,32 @@
 	<version>1.0-SNAPSHOT</version>
 	<name>com.musala.sdcs</name>
 
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>1.5.2.RELEASE</version>
+	</parent>
+
 	<dependencies>
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>3.8.1</version>
-			<scope>test</scope>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+
 		<dependency>
 			<groupId>mysql</groupId>
 			<artifactId>mysql-connector-java</artifactId>
-			<version>5.1.39</version>
-		</dependency>
-		<dependency>
-			<groupId>org.slf4j</groupId>
```

Comment:
Furthermore, running `mvn clean install` cannot build the project due to compilation failures. In Eclipse, `org.slf4j.*` cannot be resolved, as well as ` org.springframework.*`. Please resolve all dependencies.